### DISCUSSION TITLE
slow Ultimate SD upscale in SD.Next, tile artifacts abound; can't find MultiDiffusion among extensions

### DISCUSSION THREAD

**Discussion by minsky91** [vladmandic/sdnext#3150] [Q&A]
I use Ultimate upscale and SD upscale extensively, as well as MultiDiffusion, both in Automatic and Forge. In the description of Ultimate upscale in SD.Next I read that it has 'more advanced options' and 'less artifacts than original using higher denoise ratio 0.3-0.5'.  I have just ran a Ultimate SD upscale for a 7680x4320 image with 0.4 denoise and Scale 1 (just a refining pass), and the result was heavy with visible tile artifacts, much worse than I had it under Forge or Automatic.  It also took considerably longer: 21 min. against 10 min. in Automatic and 4 minutes under Forge, with the same file and default parameters. Maybe I am missing some optimization parameters?
... Could SD upscale be a better choice under SD.Next, than the Ultimate one?

Another related Q: I have MultiDiffusion extension installed in SD.Next, but it doesn't show up in extensions. Where do I find it??

**Comment by xomstor** [comment]
MultiDiffusion won't show for me too, may some Cursor or Claude can help integrate that..

**Comment by vladmandic** [comment]
both multidiffusion and sd ultimate upscale have not been updated in over 2 years which is forever in ai/ml world.
multidiffusion has not been compatible with sdnext for a long time and given its internal architecture, it cant.
sd ultimate upscale is in reality a simple script. does it work or not, or how well, dont know, but there is nothing structurally preventing it.
